app/leagues/page.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  LeagueCard,
+  LeagueCardHeader,
+  LeagueCardTitle,
+  LeagueCardDescription,
+  LeagueCardContent,
+} from '@/components/LeagueCard/LeagueCard';
+
+export default function Leagues() {
+  return (
+    <div>
+      <h1 className="pb-10 text-center text-3xl font-bold leading-[-3%]">
```

Comment:
We should create our own heading components. This way we have the same styles for all heading across the board.

Reply:
@shashilo you mean headings as in `h1` - `h6`? Why not just outline those styles in globals.css? I tend to do this [in my own projects](https://github.com/ryandotfurrer/ryanfurrer-v5/blob/e5dd04fdf0996517d38e32257aeaf3d129c77e84/src/styles/global.css#L35-L66) and [update the default tailwind](https://github.com/ryandotfurrer/ryanfurrer-v5/blob/e5dd04fdf0996517d38e32257aeaf3d129c77e84/tailwind.config.cjs#L13-L22) in `tailwind.config.cjs`

Let me know which you prefer and I'll get it done.

Reply:
Opened #304 for this effort.